expo/expo
# [docs] [filesystem] Remove mentions of deprecated CameraRoll API

# Why

Since CameraRoll API from core React Native has been gone for quite some time now, it made sense to remove it from the docs of the `expo-file-system` package as well to avoid any potential confusion. This is what this PR is about.



# How

Remove mentions of deprecated CameraRoll API in the `expo-file-system` docs.



# Test Plan

- run the docs project
- check the SDK44 and unversioned docs of the FileSystem package



# Checklist




P.S. This is my first PR here, please let me know if there's anything that should be improved or modified! 
Thank you.
